spring-pulsar-spring-boot-autoconfigure/src/main/resources/mockito-extensions/org.mockito.plugins.MockMaker

```diff
@@ -0,0 +1 @@
+mock-maker-inline
```

Comment:
This should be under `test/resources...`

Reply:
done